packages/core/src/ui/widgets/colorBar/types/ColorBarProps.ts

```diff
@@ -0,0 +1,16 @@
+import { WidgetProps } from '../../Widget';
+import { ColorBarScalePosition } from '../enums/ColorBarScalePosition';
+import { ColorBarRange } from './ColorBarRange';
+import { ColorBarScaleStyle } from './ColorBarScaleStyle';
+import { ColorBarVOIRange } from './ColorBarVOIRange';
+import { Colormap } from './Colormap';
+
+export interface ColorBarProps extends WidgetProps {
```

Comment:
this is very similar to colorBarScaleProps, can we refactor them to extract common elements? `extends ColorBarCommonProps`